client/components/mma/holiday/HolidayReview.tsx

```diff
@@ -253,7 +253,11 @@ export const HolidayReview = () => {
 						{!!productType.holidayStops
 							.explicitConfirmationRequired &&
 							!isCheckboxConfirmed && (
-								<InlineError>
+								<InlineError
+									cssOverrides={css`
+										margin-top: ${space[3]}px;
+									`}
+								>
```

Comment:
again just fixing a very small layout issue where the inline error component was displaying inline instead of block